src/components/molecules/VisionZeroImageViewWidget.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FC } from 'react';
+import { IWidgetVisionZeroImage } from '../../models/WidgetData';
+import ImageView from './ImageView';
+
+interface IProps {
+  data: IWidgetVisionZeroImage[];
+}
+const VisionZeroImageViewWidget: FC<IProps> = (props) => {
```

Comment:
Do not make it for a specific image (vision zero) - it should be for all static images